docs/source/addons/theme.md

```diff
@@ -0,0 +1,220 @@
+@menu        : 'pastanaga';
+@message     : 'pastanaga';
+@table       : 'pastanaga';
+
+/* Modules */
+@accordion   : 'pastanaga';
+@checkbox    : 'pastanaga';
+@dimmer      : 'pastanaga';
+@dropdown    : 'pastanaga';
+@embed       : 'pastanaga';
+@modal       : 'pastanaga';
+@nag         : 'pastanaga';
+@popup       : 'pastanaga';
+@progress    : 'pastanaga';
+@rating      : 'pastanaga';
+@search      : 'pastanaga';
+@shape       : 'pastanaga';
+@sidebar     : 'pastanaga';
+@sticky      : 'pastanaga';
+@tab         : 'pastanaga';
+@transition  : 'pastanaga';
+
+/* Views */
+@ad          : 'pastanaga';
+@card        : 'pastanaga';
+@comment     : 'pastanaga';
+@feed        : 'pastanaga';
+@item        : 'pastanaga';
+@statistic   : 'pastanaga';
+
+/* Extras */
+@main        : 'pastanaga';
+@custom      : 'pastanaga';
+
+/*******************************
+            Folders
+*******************************/
+
+/* Path to theme packages */
+@themesFolder : '~volto-themes';
+
+/* Path to site override folder */
+@siteFolder  : "~<name_of_your_theme>/theme";
+
+/*******************************
+         Import Theme
+*******************************/
+
+@import (multiple) "~semantic-ui-less/theme.less";
+@fontPath : "~volto-themes/@{theme}/assets/fonts";
+
+.loadAddonOverrides() {
+  @import (optional) "@{siteFolder}/@{addon}/@{addontype}s/@{addonelement}.overrides";
+}
+
+/* End Config */
+```
+
+3. Declare the theme as an add-on by adding its name as the value for the `addons` key in either `volto.config.js` or `package.json`.
+4. After starting Volto, the theme should be active.
+   Now you can add overrides to the default theme in `src/theme`, same as you would in a project.
+5. Now you can safely delete your project's `theme` folder, since the one in the add-on will take precedence.
+
+## Using your own escape hatch
+
+Volto theming uses SemanticUI theming capabilities to define and extend a theme for your site.
+However, while maintaining and playing well with the Semantic UI Volto base, using a traditional CSS approach can be done using the Less-based `extras` escape hatch.
+
+At the same time, one can entirely discard the extras escape hatch and add your own, by customizing the `theme.js` module in Volto.
+
+```js
+import 'semantic-ui-less/semantic.less';
+import '@plone/volto/../theme/themes/pastanaga/extras/extras.less';
+
+// You can add more entry points for theming
+import '@kitconcept/volto-light-theme/theme/main.scss';
+```
+
+Customizing it is a special use case  in Volto: add a `./@root/theme.js` file structure in your `customizations` folder in your add-on or project.
+
+While building your own escape hatch for theming, you can use the preprocessor of your choice (in the example, SCSS) while maintaining the "base" Volto theme, but customizing it using the resultant CSS.
+
+You can see an example of such a theme in: https://github.com/kitconcept/volto-light-theme
+
+## Modify a custom theme from another add-on
```

Comment:
This is about how to write a theme add-on, so that variables can be overridden to customize the base theme with minimal effort, am I correct?
And if I want to work with variables, I have to use scss instead of less, correct?
But in simple cases I can still write a simple theme add-on, use it in a project, and just add some CSS rules to for example make the breadcrumbs background light green, do I?

Reply:
> This is about how to write a theme add-on, so that variables can be overridden to customize the base theme with minimal effort, am I correct?

Correct. Not only from one place, but from any add-on, since the entrypoint collect them all in the order the add-ons are specified in the add-ons list.

> And if I want to work with variables, I have to use scss instead of less, correct?

The implementation is based on a convention and under the same type of pre-processor. In this case, the conventions are a variable and main files (which will be collected from all add-ons) and the pre-processor is scss. Ideally it should be possible do the same for less, and any other file conventions... 

> But in simple cases I can still write a simple theme add-on, use it in a project, and just add some CSS rules to for example make the breadcrumbs background light green, do I?

Yes, as always... this does not replace anything. What this enables, given a proper SCSS based theme as the one in the example, is that any add-on could "sum up" to the theme.